src/core/project/qgsproject.h

```diff
@@ -1362,6 +1362,22 @@ class CORE_EXPORT QgsProject : public QObject, public QgsExpressionContextGenera
      */
     bool removeAttachedFile( const QString &path );
 
+    /**
+     * Returns an identifier for an attachment file path
```

Comment:
Would be nice to give a brief explanation what an "identifier" is for someone not familiar with it.